milestone-3/frontend/src/components/recipe_page.jsx

```diff
@@ -0,0 +1,87 @@
+import './recipe_page.css';
+import Navbar from './navbar.jsx';
+import React, { useState } from 'react';
+
+
+function Recipe_page(prop) {
+
+  //states for the ingrediants and instructions in list
+  const [newIngredient, setNewIngredient] = React.useState('');
+  const [newInstruction, setNewInstruction] = React.useState('');
+
+  //functions that will add the input into the javaS object and then update the html
+  let addIngr = () =>{
+      prop.recipe.ingrediantList.push(newIngredient);
```

Comment:
Although this will technically work, its typically considered  a poor practice for a child to modify its own props in React. I'd recommend either using your milestone 2 code or storing the list of ingredients in a state variable and modifying that instead.